module_utils/network/junos/argspec/facts/facts.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The arg spec for the junos facts module.
+"""
+
+
+class FactsArgs(object):  # pylint: disable=R0903
+    """ The arg spec for the junos facts module
+    """
+
+    def __init__(self, **kwargs):
+        pass
+
+    choices = [
+        'all',
+        'l3_interfaces',
+    ]
+
+    argument_spec = {
+        'gather_subset': dict(default=['!config'], type='list'),
+        'gather_network_resources': dict(default=['all'],
```

Comment:
the default value is not required for `gather_network_resources`

Reply:
This just came from the rmb output, I assume it has been updated